### PR TITLE
Fix Kyverno deployment name check

### PR DESCRIPTION
**Pull request by dhaiducek**
Kyverno v1.10.0 introduced breaking changes to separate it into separate deployments

ref: https://github.com/kyverno/kyverno/releases/tag/v1.10.0

### CODE REVIEW

**Review by JustinKuli — DISMISSED** [review]
/hold

LGTM! Just one optional nit, feel free to unhold.

File: test/integration/policy_kyverno_generators_test.go [hunk 1]
```diff
@@ -88,27 +93,29 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the kyverno generator "+
 		).Should(Equal(policiesv1.Compliant))
 
 		By("Checking that kyverno deployment exists on the managed cluster")
```
[the remark this answers is not in the document]
Comment (dhaiducek) [reply test/integration/policy_kyverno_generators_test.go]: Since the Prow job failed anyway, I'm always down for a good grammar correction! 😄

**Review by JustinKuli — APPROVED** [review]

**Comment by dhaiducek** [comment]
Ugh. This doesn't look good. It's happened the past two times:
```
2023-06-01T16:42:58+00:00 WARN Deploy hub-1: Current MCH status is Installing. Will retry (90/900s)
E0601 16:43:13.257406    1607 memcache.go:255] couldn't get resource list for clusterview.open-cluster-management.io/v1alpha1: the server is currently unable to handle the request
E0601 16:43:13.260101    1607 memcache.go:255] couldn't get resource list for proxy.open-cluster-management.io/v1beta1: the server is currently unable to handle the request
E0601 16:43:13.272069    1607 memcache.go:255] couldn't get resource list for clusterview.open-cluster-management.io/v1: the server is currently unable to handle the request
E0601 16:43:13.274285    1607 memcache.go:106] couldn't get resource list for clusterview.open-cluster-management.io/v1alpha1: the server is currently unable to handle the request
E0601 16:43:13.282951    1607 memcache.go:106] couldn't get resource list for proxy.open-cluster-management.io/v1beta1: the server is currently unable to handle the request
E0601 16:43:13.286460    1607 memcache.go:106] couldn't get resource list for clusterview.open-cluster-management.io/v1: the server is currently unable to handle the request
E0601 16:43:13.290719    1607 memcache.go:106] couldn't get resource list for clusterview.open-cluster-management.io/v1alpha1: the server is currently unable to handle the request
E0601 16:43:13.292184    1607 memcache.go:106] couldn't get resource list for proxy.open-cluster-management.io/v1beta1: the server is currently unable to handle the request
E0601 16:43:13.293681    1607 memcache.go:106] couldn't get resource list for clusterview.open-cluster-management.io/v1: the server is currently unable to handle the request
E0601 16:43:13.295921    1607 memcache.go:106] couldn't get resource list for clusterview.open-cluster-management.io/v1alpha1: the server is currently unable to handle the request
E0601 16:43:13.297367    1607 memcache.go:106] couldn't get resource list for proxy.open-cluster-management.io/v1beta1: the server is currently unable to handle the request
E0601 16:43:13.298833    1607 memcache.go:106] couldn't get resource list for clusterview.open-cluster-management.io/v1: the server is currently unable to handle the request
parse error: Invalid numeric literal at line 1, column 6
```
Third time's the charm, though! Fingers crossed!

**Comment by JustinKuli** [comment]
/honk

Sometimes this helps

**Comment by dhaiducek** [comment]
/unhold

...I haven't found a quick resolution to the deployment failure, but I'm pretty confident in the fix.

**Pull request merged**